RecipeCabinet.Database/dbo/StoredProcedures/Rating/Rating_Create.sql

```diff
@@ -13,8 +13,8 @@ INSERT INTO [Rating] (
 )
 OUTPUT Inserted.Id
 VALUES (
-    NOW(),
-    NOW(),
+    GETDATE(),
```

Comment:
We should consider if we want to use UTC Dates and then allow users a timezone or automatically get it via browsers

Reply:
I'll swap that logic over later. Good call.

Reply:
We don't have user data set up yet... but that will be good to consider.